ts/Dashboards/Actions/Bindings.ts

```diff
@@ -88,43 +89,49 @@ namespace Bindings {
 
     export async function addComponent(
         options: Partial<ComponentType['options']>,
-        cell?: Cell
+        cell?: Cell,
+        board?: Board
     ): Promise<(Component|void)> {
         const optionsStates = (options as any).states;
         const optionsEvents = options.events;
+        const renderTo = options.renderTo || options.cell;
 
-        cell = cell || Bindings.getCell(options.cell || '');
+        cell = cell || Bindings.getCell(renderTo || '');
 
-        if (!cell?.container || !options.type) {
+        const componentContainer =
+            cell?.container || document.querySelector('#' + renderTo);
+
+        if (!componentContainer || !options.type) {
             error(
                 `The component is misconfigured and is unable to find the
-                HTML cell element ${options.cell} to render the content.`
+                HTML cell element ${renderTo} to render the content.`
             );
             return;
         }
 
-        const componentContainer = cell.container;
-
         let ComponentClass =
             ComponentRegistry.types[options.type] as Class<ComponentType>;
 
         if (!ComponentClass) {
             error(
                 `The component's type ${options.type} does not exist.`
             );
-            ComponentClass =
-                ComponentRegistry.types['HTML'] as Class<ComponentType>;
 
-            options.title = {
-                text: cell.row.layout.board?.editMode?.lang.errorMessage,
-                className:
-                    Globals.classNamePrefix + 'component-title-error ' +
-                    Globals.classNamePrefix + 'component-title'
-            };
+            if (cell) {
```

Comment:
Can we do an early return when there is no cell? Perhaps we also should throw an error?

Reply:
What kind of error?